src/MongoDB.Driver/Linq/Linq3Implementation/LinqProviderV3.cs

```diff
@@ -113,7 +113,8 @@ internal sealed class LinqProviderV3 : LinqProvider
             IBsonSerializer<TSource> sourceSerializer,
             IBsonSerializerRegistry serializerRegistry)
         {
-            throw new NotImplementedException();
+            // TODO: implement using LINQ3 instead of falling back to LINQ2
+            return LinqProvider.V2.TranslateExpressionToFindProjection(expression, sourceSerializer, serializerRegistry);
```

Comment:
For the time being falling back on LINQ2 for this.

This isn't implemented in LINQ3 yet. This isn't LINQ per se... it's Find projection.